### PR TITLE
Uninitialized memory errors

### PR DESCRIPTION
**Pull request by bcmpinc**
Fixes two uninitialized memory errors in the toolbar.

### CODE REVIEW

File: src/toolbar.cpp [hunk 1]
```diff
@@ -154,6 +154,9 @@ bool GraphicsWindow::ToolbarDrawOrHitTest(int mx, int my,
 
     bool withinToolbar =
         (mx >= aleft && mx <= aright && my <= atop && my >= abot);
+    
+    // Initialize/clear menuHit.
+    if(menuHit) *menuHit = Command::NONE;
```
Comment (whitequark) [inline src/toolbar.cpp]: You've moved this above the early return. Are you sure this is sound?

Comment (whitequark) [reply src/toolbar.cpp]: Nevermind, I misunderstood.

**Comment by whitequark** [comment]
Thanks!

File: src/toolbar.cpp (hunk 1 above)
Comment (bcmpinc) [reply src/toolbar.cpp]: Yes, it is the early return that triggers the uninitialized memory error.

**Pull request merged**